### PR TITLE
Improve coding standards

### PR DESCRIPTION
**Pull request by schmengler**

### CODE REVIEW

**Comment by schmengler** [comment]
Possible duplicate of #1 :thinking:

File: Exception/NotImplementedException.php [hunk 1]
```diff
@@ -11,4 +11,8 @@
  */
 class NotImplementedException extends Exception
 {
+    public function getMessage(): string
+    {
+        return 'Where is your god now?';
```
Comment (jissereitsma) [inline Exception/NotImplementedException.php:16]: Good point. This is a quality improvement!

**Comment by jissereitsma** [comment]
Please split up multiple changes in multiple PRs. Now it is impossible to merge the changes as a whole, because some improvements are agreed upon and others are not. I'll borrow the idea for a better exception message right away!

**Comment by jissereitsma** [comment]
https://github.com/yireo-training/magento2-mage-god/commit/2fe24cb42ea4f85123cda02d53795a1cc7f78c3f#diff-539599c8028a41a3a729e5e4a442f507

**Pull request closed without merging**